recipes/temporalio-cli/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  version: "1.3.0"
+
+package:
+  name: temporalio-cli
+  version: ${{ version }}
+
+source:
+  url: https://github.com/temporalio/cli/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 15be9f155cd5114367942568f884969f7ed2d3262ad39bb665cf359735f643b3
+  target_directory: src
+
+build:
+  number: 0
+  script:
+    - cd src
+    - go-licenses save ./cmd/temporal --save_path ../library_licenses --ignore github.com/cactus/go-statsd-client/statsd --ignore modernc.org/mathutil
+    - if: win
+      then: go build -ldflags="-s -w -X github.com/temporalio/cli/temporalcli.Version=${{ version }}" -v -o %LIBRARY_BIN%\temporal.exe ./cmd/temporal
+      else: go build -ldflags="-s -w -X github.com/temporalio/cli/temporalcli.Version=${{ version }}" -v -o $PREFIX/bin/temporal ./cmd/temporal
+    - if: unix
+      then:
+        - mkdir -p $PREFIX/share/bash-completion/completions
+        - temporal completion bash > $PREFIX/share/bash-completion/completions/temporal
+        - mkdir -p $PREFIX/share/zsh/site-functions
+        - temporal completion zsh > $PREFIX/share/zsh/site-functions/_temporal
+        - mkdir -p $PREFIX/share/fish/vendor_completions.d
+        - temporal completion fish > $PREFIX/share/fish/vendor_completions.d/temporal.fish
```

Comment:
how will this work when cross-compiling from osx-64 to osx-arm64?

Reply:
That is a good question. It won't work well, if we are not also building a `osx-64` binary at the same time (only to generate the completions).

The other idea would be to create a `noarch` package that only contains the completions, and skip that when cross compiling but that might also not be the ideal solution.

Reply:
As discussed, we’ll do native builds for this one so it should be fine